app/com/linkedin/drelephant/exceptions/TonYExceptionFingerprinting.java

```diff
@@ -55,9 +56,13 @@
   private HashSet<String> exceptionIdSet = new HashSet<>();
   @Getter
   private List<ExceptionInfo> _exceptionInfoList = new ArrayList<>();
+  private String className = null;
```

Comment:
`className` seems ambiguous, can rename it to `exceptionCategoryName` or else more appropriate name